SwiftDashboard.md

```diff
@@ -45,6 +46,7 @@ for Storage.
 Property Wrappers and likely the SwiftUI lifecycle bits, but an audit and improvements could likely be made. The existing FIAM and Analytics View modifier
 APIs would fit into this category.
 * **Property Wrappers**: Property wrappers are used to improve the API.
+* **Swift Doc Scrub**: Review and update to change Objective C types and call examples to Swift.
```

Comment:
Nice! Do you think it's worth adding something about ensuring future support for DocC in our formatting?

Reply:
Good idea! Added a sentence.